bounties/php/CMS/1/vulnerability.json

```diff
@@ -0,0 +1,52 @@
+{
+    "PackageVulnerabilityID": "1-Lack of rate limiting",
+    "DisclosureDate": "Aug 14",
+    "AffectedVersionRange": "< = 5.8.0",
```

Comment:
**As it's the latest version just put a "*" on the `AffectedVersionRange` key.**

```json
"AffectedVersionRange": "< = 5.8.0"
```

**to** :point_down: 

```json
"AffectedVersionRange": "*"
```